File: classes/parte-6-datos-y-estructuras/099-registros-structs-y-clases/implementaciones/c/main.c
#include <stdio.h>

struct Persona {
    char nombre[64];
    long edad;
};

int main(void) {
    struct Persona p;
    if (scanf("%63s %ld", p.nombre, &p.edad) != 2) return 1;
    printf("Persona(nombre=%s, edad=%ld)\n", p.nombre, p.edad);
    return 0;
}

File: classes/parte-6-datos-y-estructuras/099-registros-structs-y-clases/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

type Persona struct {
	Nombre string
	Edad   int
}

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	t := strings.Fields(line)
	edad, _ := strconv.Atoi(t[1])
	p := Persona{Nombre: t[0], Edad: edad}
	fmt.Printf("Persona(nombre=%s, edad=%d)\n", p.Nombre, p.Edad)
}
